react-components/src/data-providers/core-dm-provider/filterNodesByMappedTo3d.ts

```diff
@@ -74,7 +79,27 @@ function createRelevantObject3dKeys(
       )
     );
 
-  return new Set<FdmKey>([...cadObject3dList, ...pointCloudObject3dList]);
+  const relevant360NodeKeys = new Set<FdmKey>(
+    [
+      ...connectionData.items.initial_360_image_nodes,
+      ...connectionData.items.direct_360_image_nodes,
+      ...connectionData.items.indirect_360_image_nodes
+    ].map(createFdmKey)
+  );
+
+  const relevant360AnnotationEdges = (
+    [
+      ...connectionData.items.initial_360_annotation_edges,
+      ...connectionData.items.direct_360_annotation_edges,
+      ...connectionData.items.indirect_360_annotation_edges
+    ] as unknown[] as EdgeItem[]
```

Comment:
Can we avoid `as unknown as X` here? It seem to be the incorrect type:
```
is missing the following properties from type 'EdgeItem': startNode, endNode
```

Reply:
It took an unjustifiable amount of time, but I've managed to get rid of this cast